rust-engine/src/printer/pretty_ast.rs

```diff
@@ -59,11 +59,14 @@ pub struct DebugJSON<T: serde::Serialize>(pub T);
 
 impl<T: serde::Serialize> Display for DebugJSON<T> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        const PATH: &str = "/tmp/hax-ast-debug.json";
```

Comment:
This assumes that there's a `/tmp`, so won't build on something like Windows. Maybe use `temp_dir` instead?

Reply:
Sure!

Reply:
Maybe I can put that in `$HOME/.hax/debug.json`, wdyt?

Reply:
Ok, spending more time on that is silly.
This is a printer dev facility: I simply added a `cfg(unix)`, and a default implementation for non-unix targets.

Reply:
👍 https://github.com/cryspen/hax/issues/1616